docs/install.md

```diff
@@ -70,6 +70,7 @@ to get started.
 
     $ pip install docker-compose
 
+> **Note:** pip version 6.0 or greater is required
```

Comment:
I would agree with you that it's not semantically correct. However this is the convention we use all over in the docs (not just in compose, but all the docker projects).

I think the reason is that markdown is so limited, it doesn't provide any other way to create a note like this with the proper style (something like http://sphinx-doc.org/markup/para.html#directive-note).

Yet another reason I think markdown is incomplete for documentation purposes!